src/full/Agda/TypeChecking/Monad/Base.hs

```diff
@@ -4344,6 +4344,7 @@ data Warning
   | RewriteMissingRule Term Term Term
     -- ^ The global confluence checker found a term @u@ that reduces
     --   to @v@, but @v@ does not reduce to @rho(u)@.
+  | RewriteLHSReducesTo QName Term Term
```

Comment:
I suggest to drop the `To`:  `RewriteLHSReduces`.  
(Otherwise, it looks weird in `-W` flag.)